client/components/wcc-settings-form/index.js

```diff
@@ -9,19 +9,76 @@ import { successNotice, errorNotice } from 'state/notices/actions';
 import * as SettingsActions from 'state/settings/actions';
 import * as PackagesActions from 'state/form/packages/actions';
 import { getFormErrors } from 'state/selectors';
+import { translate as __ } from 'lib/mixins/i18n';
+import FormButton from 'components/forms/form-button';
+import FormButtonsBar from 'components/forms/form-buttons-bar';
 
 const WCCSettingsForm = ( props ) => {
+	const currentStep = props.form.currentStep;
+	const currentStepLayout = props.layout[ currentStep ];
+
+	const renderCurrentStep = () => {
+		if ( ! currentStepLayout ) {
+			return <span>... spinner or something ...</span>;
```

Comment:
For now, can we replace `... spinner or something ...` with an actual spinner, or a gridicon.. something?